### PR TITLE
Log In and Sign In UI ready for firebase conexion and authentication

### PR DESCRIPTION
**Pull request by mariam-rivera**
# Description
We already done the Sign In and Login In UI, for the firebase conexion and authentication. 
 - We added forms to capture user data (Log In and Sign In)
 - We re-use code in the components Form Input and Form Button
 - We structured our code with Prettier
 - Navigation still working successfully
 - We tested every component with snapshoot testing
 - Negative values fixed
 - useState names fixed 

In the next version we will update the images added to vector icons, of course!👍 
 
  ## Screenshots 

image

### CODE REVIEW

**Review by fernandoamz — CHANGES_REQUESTED** [review]
Create the snapshoot testing for every component

File: ReservacionDeVuelos/App.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
...
 
 const App = () => {
-  return <Providers/>;
-}
+  return <Providers />;
+};
 
-export default App;
\ No newline at end of file
+export default App;
```
Comment (fernandoamz) [inline ReservacionDeVuelos/App.js:8]:
```suggestion
export default App;

```

File: ReservacionDeVuelos/src/components/Checkbox.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      </View>
+    </>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    marginHorizontal: 2,
+    marginTop: 15,
+    marginLeft: -18,
```
Comment (fernandoamz) [inline ReservacionDeVuelos/src/components/Checkbox.js]: Why this value is negative?

**Review by fernandoamz — CHANGES_REQUESTED** [review]

**Review by hearagmu — DISMISSED** [review]
Good!

File: ReservacionDeVuelos/src/components/Checkbox.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            value={isSelected}
+            onValueChange={setSelection}
+            style={styles.checkbox}
+            tintColors={{true: colors.blue, false: colors.black}}
+          />
+          <Text style={styles.labelCheckBox}>
+            I agree to the
+            <Text style={{textDecorationLine: 'underline'}}> Terms</Text> and
+            <Text style={{textDecorationLine: 'underline'}}>
+              {' '}
```
Comment (LeyaimMagma) [inline ReservacionDeVuelos/src/components/Checkbox.js]: Is this necessary?

File: ReservacionDeVuelos/src/components/Checkbox.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
+import React, {useState} from 'react';
+import {StyleSheet, View, Text} from 'react-native';
+import colors from '../utils/Colors';
+import CheckBox from '@react-native-community/checkbox';
+import contentText from '../utils/Constants';
+
+function CheckBoxComponent() {
+  const [isSelected, setSelection] = useState(false);
+  const [isSelected2, setSelection2] = useState(false);
```
Comment (LeyaimMagma) [inline ReservacionDeVuelos/src/components/Checkbox.js]: these are bad names

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

File: ReservacionDeVuelos/src/components/Checkbox.js (hunk 3 above)
Comment (LeyaimMagma) [reply ReservacionDeVuelos/src/components/Checkbox.js]: `{' '}`

**Review by Ing-Heber — COMMENTED** [review]
Good job, please apply what leyaim told you (give significant names so we can understand easily what it does) 😅

**Review by ChantalRamirez — APPROVED** [review]
Good job, but the names of the constant in your usestate is incorrect. 
Example: const [Selected, setSelected] = useState(false);

**Review by GracielaJazmin — DISMISSED** [review]

**Review by ChantalRamirez — DISMISSED** [review]
Good

File: ReservacionDeVuelos/src/components/Checkbox.js (hunk 4 above)
Comment (JordiGN) [reply ReservacionDeVuelos/src/components/Checkbox.js]: Try to be more specific on what component/value does that state affects, e.g "inputNameIsSelected, setInputNameSelection"
"inputEmailIsSelected, setInputEmailSelection"

**Review by JordiGN — DISMISSED** [review]
Nice job, just a few pieces of advice and details!

**Review by hearagmu — DISMISSED** [review]
Good!

File: ReservacionDeVuelos/src/components/FormInput.js [hunk 5]
```diff
@@ -1,12 +1,79 @@
...
+      <View style={styles.inputContainer}>
+        <TextInput
+          value={labelValue}
+          style={styles.input}
+          placeholder={placeholderText}
+          secureTextEntry={secureTextEntry}
+        />
+      </View>
+      <View style={styles.containerIconEye}>
+        {ruta != undefined && (
```
Comment (LeyaimMagma) [inline ReservacionDeVuelos/src/components/FormInput.js]: Use `{ ruta &&  <Touch....} `. this validate if `ruta` is true

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

File: ReservacionDeVuelos/src/components/FormInput.js (hunk 5 above)
Comment (LeyaimMagma) [reply ReservacionDeVuelos/src/components/FormInput.js]: also it validates different of undefined or null

**Review by GracielaJazmin — DISMISSED** [review]

**Review by LeyaimMagma — APPROVED** [review]

**Review by hearagmu — APPROVED** [review]
ok

File: ReservacionDeVuelos/src/screens/SignInScreen.js [hunk 6]
```diff
@@ -1,34 +1,123 @@
...
+import FormButton from '../components/FormButton';
+import FormInput from '../components/FormInput';
+import GoogleButton from '../components/GoogleButton';
+import Checkbox from '../components/Checkbox';
+import eye from '../img/ojo.png';
 
 const SignInScreen = ({navigation}) => {
+  const [name, setName] = useState();
+  const [email, setEmail] = useState();
+  const [password, setPasword] = useState();
```
Comment (fernandoamz) [inline ReservacionDeVuelos/src/screens/SignInScreen.js:14]:
```suggestion
  const [name, setName] = useState('');
  const [email, setEmail] = useState('');
  const [password, setPasword] = useState('');
```

Always define the value type on useState

**Review by Ing-Heber — APPROVED** [review]

**Review by fernandoamz — APPROVED** [review]
LGTM!

**Pull request merged**